tools/vsimporter/src/query.cpp

```diff
@@ -76,7 +76,7 @@ static bool parseListSelection(const String& selection, size_t upperBound, std::
 
 void queryListSelection(const StringVec& values, const String& queryMessage, const String& itemName, std::vector<size_t>& ret)
 {
-  sbAssert(!values.empty());
+  sbAssertWithTelemetry(!values.empty(), "Unable to query empty list of values");
```

Comment:
I moved the assert to the calling functions.

---
In reply to: [82420134](https://github.com/Microsoft/WinObjC/pull/1104#discussion_r82420134) [](ancestors = 82420134)